compiler/ilgen/OMRBytecodeBuilder.cpp

```diff
@@ -41,31 +41,18 @@ OMR::BytecodeBuilder::BytecodeBuilder(TR::MethodBuilder *methodBuilder,
                                       int32_t bcLength)
    : TR::IlBuilder(methodBuilder, methodBuilder->typeDictionary()),
    _fallThroughBuilder(0),
-   _bcIndex(bcIndex),
    _name(name),
    _bcLength(bcLength),
    _initialVMState(0),
    _vmState(0)
    {
+   _bcIndex = bcIndex;
```

Comment:
Wouldn't it be better to pass `bcIndex` to `TR:ILBuilder`'s constructor? As it stands, if another class extends `TR::IlBuilder`, it would have to remember to initialize `_bcIndex`.

Reply:
Yep. Good catch, I was cutting corners there. I'll add it to IlBuilder's constructor.